addons/mail/static/tests/clock_model_tests.js

```diff
@@ -0,0 +1,42 @@
+/** @odoo-module **/
+
+import { insertAndReplace } from '@mail/model/model_field_command';
+import { afterEach, beforeEach, start } from '@mail/utils/test_utils';
+
+QUnit.module('mail', {}, function () {
+QUnit.module('clock_model_tests', {
+    async beforeEach() {
+        await beforeEach(this);
+
+        this.start = async params => {
+            const res = await start({ ...params, data: this.data });
+            const { apps, env, widget } = res;
+            this.apps = apps;
+            this.env = env;
+            this.widget = widget;
+            return res;
+        };
+    },
+    afterEach() {
+        afterEach(this);
+    },
+});
+
+QUnit.test('A clock is deleted when it has no more watcher.', async function (assert) {
+    assert.expect(1);
+
+    await this.start();
+    const watcher = this.messaging.models['ClockWatcher'].insert({
+        qunitTestOwner: insertAndReplace(),
+        clock: insertAndReplace({ frequency: 180 * 1000 }),
+    });
+
+    watcher.delete();
+    assert.strictEqual(
+        this.messaging.models['Clock'].all().length,
+        0,
+        "deleting all the watchers of a clock should result in the deletion of the clock itself."
```

Comment:
single assert test should ideally have assert comment equals test title (not sure which one is best in this case, maybe a mix of both)